src/Korobi/WebBundle/Util/GitInfo.php

```diff
@@ -13,45 +13,130 @@ class GitInfo {
 
     protected $branch;
     protected $hash;
+    protected $tempRef;
 
     /**
-     * @param $environment
-     * @param $rootDir
+     * @param string $environment Current environment's name.
+     * @param string $appDir Absolute path to app directory.
      */
-    public function __construct($environment, $rootDir) {
-        $this->updateData($rootDir, $environment);
+    public function __construct($environment, $appDir) {
+        $this->updateData(dirname($appDir), $environment);
     }
 
     /**
      * Force update data.
      *
-     * @param string $environment
+     * @param string $rootDir Absolute path to root directory.
+     * @param string $environment Current environment's name.
      */
-    public function updateData($rootDir, $environment = 'dev') {
-        // Fast information retrieval for online environments
-        if (in_array($environment, self::ONLINE_ENVS)) {
-            if ($environment == 'prod') {
-                $this->branch = 'www1-stable';
+    public function updateData($rootDir, $environment) {
```

Comment:
it's the kernel dir injected by symfony. I don't think you can inject the root dir directly